mmaction/core/evaluation/eval_detection.py

```diff
@@ -0,0 +1,249 @@
+import json
+
+import numpy as np
+import pandas as pd
+from joblib import Parallel, delayed
+
+from .accuracy import interpolated_prec_rec, segment_iou
+
+
+class ANETdetection(object):
+
+    def __init__(self,
+                 ground_truth_filename=None,
+                 prediction_filename=None,
+                 tiou_thresholds=np.linspace(0.5, 0.95, 10),
+                 verbose=False):
+        if not ground_truth_filename:
+            raise IOError('Please input a valid ground truth file.')
```

Comment:
FileNotFoundError

Reply:
Guess FileNotFoundError should be used when `not os.path.exists(filename)`. Here we just check if the user provides a value as the argument